kcidb/misc.py

```diff
@@ -106,6 +107,10 @@ def __init__(self, *args, **kwargs):
             kwargs: Keyword arguments to initialize ArgumentParser with.
         """
         super().__init__(*args, **kwargs)
+        self.add_argument(
+        '-v', '--version',
+         action='version',                    
+        version="Version % s" % importlib.metadata.version('kcidb'))
```

Comment:
Could you please use f-strings for formatting the output?
I'm trying to keep that as a convention in KCIDB.